rskj-core/src/main/java/co/rsk/core/bc/BlockChainFlusher.java

```diff
@@ -100,7 +100,7 @@ private synchronized void flush() {
     }
 
     private void flushAll() {
-        Metric metric = profiler.start(Profiler.PROFILING_TYPE.BLOCKCHAIN_FLUSH);
+        Metric metric = profiler.start(Profiler.MetricKind.BLOCKCHAIN_FLUSH);
```

Comment:
Why are we changing `PROFILING_TYPE` to `MetricKind`?

Can we add the the reasoning in the PR description?

Reply:
the name itself was changed to reflect its meaning - it's type of metric, not type of profiling. In addition to that it was changed from upper snake case to pascal one to align with other enums naming - Profiler.PROFILING_TYPE isn't an enum value, it's an enum name

Reply:
Sounds good, please add that to the PR description. Thanks